go.mod

```diff
@@ -2,6 +2,10 @@ module github.com/ethereum/go-ethereum
 
 go 1.20
 
+replace github.com/specularL2/specular/lib/el_golang_lib => ../../../lib/el_golang_lib
+
+replace github.com/specularL2/specular/ops => ../../../ops
```

Comment:
where is `ops` used? or is it required bc of `el_golang_lib`?

Reply:
yeah, `el_golang_lib` uses ops to get the oracle storage slots